integration-tests/spring-data-jpa/src/main/java/io/quarkus/it/spring/data/jpa/CarRepository.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.it.spring.data.jpa;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CarRepository extends JpaRepository<Car, Integer> {
```

Comment:
Can you also add a custom method here and then test it as well?

Reply:
@geoand Hi Georgios!
Do you mean something like this?